vscode/install.bash

```diff
@@ -1,8 +1,27 @@
 #! /usr/bin/env bash
 
-if [ ! -f /etc/apt/sources.list.d/vscode.list ]
+tue-install-snap-now code
+
+tue-install-system-now default-jre
+
+# Install ros extension
+code --install-extension ms-iot.vscode-ros
+
+# Configure workspace
+workspacefile="${TUE_SYSTEM_DIR}/.vscode/settings.json"
+mkdir -p "${TUE_SYSTEM_DIR}/.vscode"
+if [[ ! -f "${workspacefile}" ]]
+then
+    tue-install-cp workspace_settings.json "${workspacefile}"
+else
+    tue-install-system-now jq
+    jq -s add "${workspacefile}" "${TUE_INSTALL_CURRENT_TARGET_DIR}/workspace_settings.json" > "${workspacefile}.new"
+    $? && mv "${workspacefile}.new" "${workspacefile}"  # Doesn't work at once above
+fi
+
+# Install and configure catkin tools extension
+code --install-extension betwo.b2-catkin-tools
```

Comment:
Should all of this be executed on each run? Or better, is it safe to execute this on all runs? I think `mkdir` will produce a warning/error in case a folder already exists.

Reply:
`mkdir -p` doesn't

I could not come up with a checker for vscode that was faster than installing it again.